### DISCUSSION TITLE
v0.90

### DISCUSSION THREAD

**Discussion by DjangoCRM** [DjangoCRM/django-crm#2] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/DjangoCRM/django-crm/releases/tag/v0.90'>v0.90</a>.</em>

**Comment by Jamezslim90** [comment]
Awesome software but still trying to understand al the configurations.

**Reply by DjangoCRM** [reply]
Hi! Default settings and instructions were specified for the production server. It's quite strict.
Now the default settings are specified for running on the development server.
For the first start of the project you only need to specify the data for connection with you database. 
Please update your project from main branch. 
Do everything according to the [guide](../blob/main/docs/installation_and_configuration_guide.md).
Let me know if everything is okay now?
These changes will be reflected in the new release.
